torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -1005,6 +1019,11 @@ void ComputeAtRootDomainMapBuilder::mapAllPendingMappings(
   }
 }
 
+void ComputeAtRootDomainMapBuilder::handle(RNGOp* rop) {
+  const TensorDomain* td = rop->output(0)->as<TensorView>()->domain();
+  mapAllPendingMappings(td);
```

Comment:
Why not just call `handle(TensorView* tv)` on `rop->output(0)->as<TensorView>()`?

Reply:
fixed